### PR TITLE
Compose, Rebase Playbooks and create VM script

### PR DESCRIPTION
**Pull request by trishnaguha**

### CODE REVIEW

File: rebase.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    command: virsh domifaddr mynode
+    register: VM_IP
+
+  - name: Create inventory file
+    file: path=inventory state=touch
+
+  - name: Write VM_IP in the inventory
+    lineinfile:
+      dest: inventory
+      line: '[atomichost]\n{{ VM_IP }} ansible_ssh_user=atomic-user ansible_ssh_pass=atomic'
```
Comment (trishnaguha) [inline rebase.yml]: @gbraad Need your help in here. I can get the IP address of the VM with the command `virsh domifaddr mynode`. But failing to write the `VM_IP` with exact IP in the inventory. The inventory looks like this:

```
[atomichost]\n{u'changed': True, u'end': u'2017-02-06 16:55:18.563692', u'stdout': u' Name       MAC address          Protocol     Address\n-------------------------------------------------------------------------------\n vnet6      52:54:00:b1:fa:fe    ipv4         192.168.121.90/24', u'cmd': [u'virsh', u'domifaddr', u'mynode'], u'start': u'2017-02-06 16:55:18.544325', u'delta': u'0:00:00.019367', u'stderr': u'', u'rc': 0, 'stdout_lines': [u' Name       MAC address          Protocol     Address', u'-------------------------------------------------------------------------------', u' vnet6      52:54:00:b1:fa:fe    ipv4         192.168.121.90/24'], u'warnings': []} ansible_ssh_user=atomic-user ansible_ssh_pass=atomic
```

**Comment by gbraad** [comment]
@trishnaguha will check this later

File: rebase.yml (hunk 1 above)
Comment (gbraad) [reply rebase.yml]: @trishnaguha can you verify the output with debug [[doc](http://docs.ansible.com/ansible/debug_module.html)]? 

Else try
```
  - name: Get the IP of VM
    command: virsh domifaddr mynode | tail -2 | awk '{print $4}' | cut -d/ -f1
    register: VM_IP

  - copy:
    dest=inventory
    content="[atomichost]\n{{ VM_IP }} ansible_ssh_user=atomic-user ansible_ssh_pass=atomic"
```

Note: "Strip the first 2 lines, get the fourth column, cut all after /". Issue remains when the VM has more than one interface.

Comment (trishnaguha) [reply rebase.yml]: Ahh..Yeah I didn't process the output of VM_IP.
I can see that I am getting the right response, just needs processing.

```
ok: [localhost] => {
    "msg": "{u'changed': True, u'end': u'2017-02-06 18:00:23.176018', u'stdout': u' Name       MAC address          Protocol     Address\\n-------------------------------------------------------------------------------\\n vnet7      52:54:00:bc:23:0c    ipv4         192.168.121.102/24', u'cmd': [u'virsh', u'domifaddr', u'mynode'], u'start': u'2017-02-06 18:00:23.157048', u'delta': u'0:00:00.018970', u'stderr': u'', u'rc': 0, 'stdout_lines': [u' Name       MAC address          Protocol     Address', u'-------------------------------------------------------------------------------', u' vnet7      52:54:00:bc:23:0c    ipv4         192.168.121.102/24'], u'warnings': []}.stdout"
}
```

Comment (gbraad) [reply rebase.yml]: @trishnaguha I updated my previous response with a hint/solution. But nicer regex are possible... and also, you still have the execution state information from Ansible in there ;-)

**Comment by trishnaguha** [comment]
@gbraad Updated the PR with changes. Going to add README file for better understanding.

**Comment by gbraad** [comment]
LGTM... seems you pushed ;-). Closing #1

**Review by gbraad — APPROVED** [review]

**Comment by trishnaguha** [comment]
Heh! Thanks :-).

**Comment by gbraad** [comment]
Glad you added the cloud-init instructions

**Comment by trishnaguha** [comment]
@gbraad I got it. I should use `shell` module instead of `command` module. Since there is `--` in the command we are issuing. Sorry for the noise ;).

File: create-vm.sh [hunk 2]
```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+set -euo pipefail
+
```
Comment (jlebon) [inline create-vm.sh:3]: Could you include a link back to http://jlebon.com/devconf/slides.pdf here?
I would appreciate the attribution. :)

Comment (trishnaguha) [reply create-vm.sh:3]: @jlebon Absolutely. I have added that in the blog post I'm doing for fedora magazine and projectatomic. Certainly will add the same here :). Thanks for your workshop and the script. Got the inspiration from you to work on this.

Comment (jlebon) [reply create-vm.sh:3]: @trishnaguha Awesome, I'm glad you liked it! :)

**Pull request merged**